ts/features/bonus/bpd/components/superCashbackRanking/SuperCashbackRanking.tsx

```diff
@@ -0,0 +1,159 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import { View } from "native-base";
+import { fromNullable } from "fp-ts/lib/Option";
+import { GlobalState } from "../../../../../store/reducers/types";
+import { bpdSelectedPeriodSelector } from "../../store/reducers/details/selectedPeriod";
+import ItemSeparatorComponent from "../../../../../components/ItemSeparatorComponent";
+import { H3 } from "../../../../../components/core/typography/H3";
+import I18n from "../../../../../i18n";
+import Markdown from "../../../../../components/ui/Markdown";
+import { IOColors } from "../../../../../components/core/variables/IOColors";
+import { BpdPeriodWithInfo } from "../../store/reducers/details/periods";
+import { localeDateFormat } from "../../../../../utils/locale";
+import { FirstPositionItem } from "./FirstPositionItem";
+import { LastPositionItem } from "./LastPositionItem";
+import UserPositionItem from "./UserPositionItem";
+
+type Props = ReturnType<typeof mapStateToProps>;
+
+type tmpCitizenRankingResource = {
+  totalParticipants: number;
+  ranking: number;
+  // Numero massimo di transazioni effettuate dagli utenti che rientrano nel 'rimborso speciale'
```

Comment:
Could it be translated in english?

Reply:
Since I yet realized we merged the update to the store, we may directly remove this temporary type definition